chart/values.yaml

```diff
@@ -507,15 +536,19 @@ promlens:
 
 # Enable OpenTelemetry Operator
 # If using tobs CLI you can enable otel with --enable-opentelemetry flag
-opentelemetryOperator:
-  enabled: *otelEnabled
-  resources:
-    limits:
-      cpu: 50m
-      memory: 260Mi
-    requests:
-      cpu: 5m
-      memory: 130Mi
+opentelemetry-operator:
```

Comment:
This name change is a breaking change to our users. Could we not do it?

Reply:
I can revert it, but without it I don't know a good way of locking the otel-opterator version and pass it into the subchart.  We could have two otel sections in the `values.yaml`, but that seem just wrong...

I figured it was a breaking change and can add an upgrade path.

Reply:
Updated the upgrade doc. Please look it over and let me know if the wording needs to be changed.